app/src/main/java/org/mozilla/tv/firefox/hint/HintBinder.kt

```diff
@@ -37,12 +38,29 @@ object HintBinder {
                                 .start()
                     }
                     .subscribe { shouldDisplay ->
-                        hintContainer.isVisible = true
                         val translationY = when (shouldDisplay) {
                             true -> 0f
                             false -> hintContainer.height.toFloat()
                         }
+
                         hintContainer.animate()
```

Comment:
nit: can we add a comment somewhere in this method (maybe here or at the top) warning devs to be _very_ careful when changing this code to ensure the experiment stays disabled when A/B testing is disabled? I'm concerned the code isn't explicit enough about this (e.g. like early returning from this binding method if `ExperimentsrProvider.shouldShowHintBar` returns false).